src/map/status.cpp

```diff
@@ -10099,7 +10110,7 @@ int status_change_start(struct block_list* src, struct block_list* bl,enum sc_ty
 			break;
 
 		case SC_STONEWAIT:
-			val3 -= status_get_sc_interval(type); // Petrify time
+			val3 -= status_get_sc_interval(SC_STONE); // Petrify time
```

Comment:
Don't you need to substract the incubation time from here instead? 

For example stone curse 17+3 duration -> 5 seconds incubation time -> 15s stone duration (3% damage)
Sienna Execrate Level 2: 9+3 duration -> 0.1 seconds incubation time -> 11.9s stone duration (2% damage)